### PR TITLE
Telegram Persistent Connection Linux

### PR DESCRIPTION
**Pull request by aleff-github**

### CODE REVIEW

**Review by 13331QTR — APPROVED** [review]
B

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Telegram Persistent Connection
+
+A script used to configure a persistent connection on a Linux computer trough a pre-configured Telegram Bot.
```
Comment (dallaswinger) [inline payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md:3]: through*

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+**Category**: Execution
+
+## Description
+
+A script used to configure a persistent connesction on a Linux computer trough a pre-configured Telegram Bot.
```
Comment (dallaswinger) [inline payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md:9]: through*

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Opens a shell, download the python script trough the `curl` command outputing the file into a `connection.py` file using `-o` option, then run it and set the run of the program as a default command every times a shell is runned.
```
Comment (dallaswinger) [inline payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md:11]: through*

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Getting Started
+
+### Dependencies
+
+* Internet Connection
+
+### Settings
+
+- **Telegram Bot**: You should configure a bot trough Telegram. If you don't know how to do this, follow the guide about [Telegram Bot guide](#telegram-bot-guide). When you have create your personal Telegram Bot you should get the Telegram bot ID that you must put into the variable BOT_TOKEN at line 4 in the Python file as you can read in the line 3 comment.
```
Comment (dallaswinger) [inline payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md:23]: through*

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md (hunk 1 above)
Comment (aleff-github) [reply payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md]: Solved > d5b87f9c3781a87fcadec9d3d0a2a46e4758d12e

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md (hunk 2 above)
Comment (aleff-github) [reply payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md]: Solved > d5b87f9c3781a87fcadec9d3d0a2a46e4758d12e

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md (hunk 3 above)
Comment (aleff-github) [reply payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md]: Solved > d5b87f9c3781a87fcadec9d3d0a2a46e4758d12e

File: payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md (hunk 4 above)
Comment (aleff-github) [reply payloads/library/execution/Telegram_Persistent_Connection_Linux/README.md]: Solved > d5b87f9c3781a87fcadec9d3d0a2a46e4758d12e

**Review by dailan2 — COMMENTED** [review]
https://github.com/hak5/usbrubberducky-payloads/tree/master

**Pull request merged**